drivers/wifi/nrf700x/rpu_fw_patches.ld

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file
+ * @brief Custom Linker command/script file
+ *
+ * Custom Linker script for the Cortex-M platforms.
+ */
+
+#include <zephyr/linker/sections.h>
+#include <zephyr/devicetree.h>
+
+#include <zephyr/linker/linker-defs.h>
+#include <zephyr/linker/linker-tool.h>
+
+/*
+ * nRF53/52 series ship an external flash that can be used for XIP using QSPI/SPI.
+ *
+ * TODO: In nRF7002 external flash using SPI isn't supported by nrfjprog yet, so,
+ * the code builds but flashing fails.
```

Comment:
XIP is only possible with QSPI.

Reply:
ok, will remove nRF7002 references from this for now, till tools support is there and then we can think of implementing the same using flash API.

Reply:
So how will we implement this for nRF7002 DK (assuming the command line tools are ready) which uses SPI to talk to the external flash? Is it possible without XIP?

Reply:
Ah ok, sorry, @krish2718 you mentioned the raw flash API.